i18n/fr.json

```diff
@@ -0,0 +1,59 @@
+{
+  "menu": {
+      "exit": "QUITTER"
+    , "help": "AIDE"
+    , "completed": "FAIT"
+    , "language": "CHOISIR LA LANGUE"
+    , "cancel": "ANNULER"
+  }
+, "language": {
+      "_current": "ACTUEL"
+    , "en": "Anglais (English)"
+    , "fr": "Français"
+    , "ja": "Japonais (日本語)"
+    , "zh-cn": "Chinois (中文)"
+    , "zh-tw": "Taiwanais (國語)"
+    , "es": "Espagnol (Español)"
+    , "pt-br": "Portugais brésilien (Português)"
+  }
+, "error": {
+      "exercise": {
+          "none_active": "Aucun exercice en cours.  Choisissez-en un dans le menu."
+        , "missing": "L’exercice {{{name}}} est introuvable."
+        , "unexpected_error": "Impossible de {{mode}} : {{{err}}}"
+        , "missing_file": "ERREUR : {{{exerciseFile}}} n’existe pas !"
+        , "not_a_workshopper": "ERREUR : {{{exerciseFile}}} n’est pas un exercice d’atelier"
+        , "preparing": "Erreur à la préparation de l’exercice : {{{err}}}"
+        , "loading": "Erreur au chargement du texte de l’exercice : {{{err}}}"
+      }
+    , "no_uncomplete_left": "Il ne reste aucun exercice à faire après l’exercice actuel."
```

Comment:
"Il ne reste aucun exercice à faire après celui-ci" Repetition always sounds weird in French